content/en/docs/guides/performance.md

```diff
@@ -11,18 +11,20 @@ description: A user guide of both general and language-specific best practices t
     inactivity to allow initial RPCs to be made quickly without a delay (i.e.
     C++ channel arg GRPC_ARG_KEEPALIVE_TIME_MS). 
 
-*   **Use streaming RPCs** when wanting to avoid continuous RPC initiation,
+*   **Use streaming RPCs** when handling
+    a long-lived logical flow of data from the client-to-server,
+    server-to-client, or in both directions. Streams can avoid continuous RPC initiation,
     which includes connection load balancing at the client-side, starting a new
     HTTP/2 request at the transport layer, and invoking a user-defined method
-    handler on the server side. Streams are particularly well-suited to handling
-    a long-lived logical flow of data from the client-to-server,
-    server-to-client, or in both directions. Streams, however, cannot be load
+    handler on the server side. Streams, however, cannot be load
     balanced once they have started and can be hard to debug for stream
-    failures. \
+    failures. They also might increase performance at a small scale but can reduce 
+    scalability due to load balancing and complexity, so they should only be used 
+    when the intent is a long-lived flow of data.  \
```

Comment:
Streams do not need to be long-lived to provide benefit and unary can be long-lived. Like I mentioned before, I'd say:

> so they should only be used when they provide substantial performance or simplicity benefit to application logic. Use streams to optimize the application, not gRPC.

If we want to expand on that to make it more clear, we could give examples like "grouping related requests over time to reduce application processing per message" and "making sure a multi-part operation is handled by the same backend". But really, just "if your application benefits, consider it" seems best; I think examples would be better if we were saying cases that _didn't_ work well with streams.

Reply:
Done, thanks!